### ISSUE TITLE
Landscape rotation in fullscreen will only flip one way

### ISSUE THREAD

**Issue by Rocket-PropelledWeeb** [futo-org/grayjay-android#1313]
### What happened?

When auto-rotate is off, in fullscreen, the screen will always flip one direction. Nothing happens when I tilt the phone onto its other side. I can turn on rotation and flip it to the direction I want it then turn rotation off again, it'll stay put until I rotate my phone more than horizontal. Once I do that it will flip and get stuck the original direction again.
Previous behavior with rotation off would allow me to flip the phone and the video would flip with it in both directions.

### Grayjay Version

262

### What plugins are you seeing the problem on?

All

### When do you experience the issue?

- [ ] While logged in
- [ ] While logged out
- [X] N/A

**Comment by Argel-Tal** [comment]
Seocnding this, Samsung A32

Fullscreen in portrait mode now just expands with black bars above and below without rotating

Once I do trigger a rotate, it then forces the app into landscape mode when de-fullscreening

Only way around is turning on system level rotate. Which causes secondary problems

**Comment by SdoggaMan** [comment]
Same here on Moto Edge 30 Pro. This has changed as of the most recent update (in last 48-72 hours.) 

Using the swipe up/down to full screen, it now full-screens _only_ in it's _current_ orientation - meaning it goes into _portrait_ full screen if you're in portrait, and doesn't auto-rotate. It also doesn't go back to portrait if you swipe down _from_ a full screen video as it did previously. (This happens even if you've got portrait full screen disabled in settings.) 

Grayjay 262 / YouTube 201 / Android 13

**Comment by jmcgaughey** [comment]
Same thing is happening on my Samsung a53 5g. didn't do it until the update.

**Comment by Rocket-PropelledWeeb** [comment]
Oh interesting. Just made a clarifying discovery for the other issues reported in this thread. If you are in portrait mode and toggle fullscreen via either swiping up or hitting the button, it will go to landscape as normal. However, if you keep holding the phone upright in portrait orientation, then hit the fullscreen toggle again, it'll bring you out of fullscreen in landscape mode. If you're in fullscreen and you hold the phone in landscape mode, it acts like normal.
In short, the fullscreen button seems to trigger a rotate not from where the app currently is, but from which direction you're holding your phone up. Hope this helps.

**Comment by SdoggaMan** [comment]
I see the same in regards to landscape remaining in landscape irrespective of the orientation prior to going full screen, but mine also remains in portrait if going full screen from there. 

In essense, rotation is effectively completely disabled, almost as if the app is ONLY letting Android auto-rotate control rotation. Portrait goes portrait, landscape goes landscape, and the only way to get it to switch is to use auto-rotate. 

Notably, because it's a 'full screen' app (the correct terminology may differ!) the little pop-up rotation "suggestion" in the bottom-left doesn't appear while in the app, so you can't even use that to faux-manually switch. You have to enable auto-rotate. 

At least, that's what mine's doing!

**Comment by Rocket-PropelledWeeb** [comment]
The update that just released seems to have fixed these issues. Thanks devs!
